scania-digital-design-system/sdds
# fix(toast): fix for the size of the dismiss icon on toast

**Describe pull-request**  
Uses the correct size on the toast dismiss icon.

**Solving issue**  
Fixes: [AB#2658](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2658)

**How to test**  
1. Go to storybook link below
2. Check in Components -> Toast
3. Check the the dismiss icon is 20x20.

**Screenshots**  
-

**Additional context**  
-
